app/dependencies/rnn.py

```diff
@@ -21,10 +21,13 @@ def train_lstm(currency_name):
 
     if historical_json_data is None:
         return
-    
+
     historical_data = get_each_currency_data(historical_json_data, currency_name)
 
-    dataa = historical_data["quotes"]
+    try:
+        dataa = historical_data["quotes"]
+    except None:
```

Comment:
What is None for ?

Reply:
sometimes we don't get data from the coinmarket cap api so in that case I don't want to run the lstm model.

Reply:
Then catch the general `exception` please and remove `None`

Reply:
ok